guides/common/modules/proc_changing-the-environment-of-a-host.adoc

```diff
@@ -6,7 +6,7 @@ Use this procedure to change the environment of a host.
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
 . Click the name of the host you want to modify.
-. Click the vertical ellipsis in the *Content view details* card and select *Edit content view assignment*.
+. Click the vertical ellipsis in the *Content view environment* card and select *Edit content view assignment*.
```

Comment:
According to The IBM Style guide, we are to avoid the usage of "vertical ellipsis" when describing this UI element. See The IBM Style Guide > Technical elements > UI elements and search for "Do not use kebab icon or vertical ellipsis." You can follow the guide to find more suitable wording.

Reply:
Oh, we use that everywhere! I wonder what we're supposed to use instead..

Reply:
> Oh, we use that everywhere! I wonder what we're supposed to use instead..

Yeah I thought that was the common term from the style guide. But apparently it's icon instead of ellipsis.

Reply:
@apinnick has been looking into this, including a new Vale rule I believe, so she might be interested to chime in with more details.

Reply:
To me "Icon" seems vague. What if there are two icons and one of them is a kebab? how are you supposed to specify which one?

Reply:
Red Hat UI documentation tends to refer to the "vertical ellipsis" as an "Options menu", not an icon. [OpenShift example](https://docs.openshift.com/container-platform/4.17/security/cert_manager_operator/cert-manager-operator-uninstall.html#cert-manager-remove-resources-console_cert-manager-operator-uninstall):

![image](https://github.com/user-attachments/assets/9b4857ff-8d7a-4e2b-8192-ae6f5f73cd66)

Without checking the UI to see what a 'card' looks like, I can't say exactly on how this step should be rewritten. It is likely to be this: "Click the options menu [icon_image] in the *Content view environment* card and select *Edit content view assignment*.

Reply:
Update: Looks like we also call it the "options icon".